osu.Game.Rulesets.Osu/Objects/Drawables/Pieces/DefaultSpinnerDisc.cs

```diff
@@ -93,7 +92,7 @@ protected override void LoadComplete()
             base.LoadComplete();
 
             drawableSpinner.RotationTracker.Complete.BindValueChanged(complete => updateComplete(complete.NewValue, 200));
-            drawableSpinner.State.BindValueChanged(updateStateTransforms, true);
+            drawableSpinner.ApplyCustomUpdateState += updateStateTransforms;
```

Comment:
These all need to have unbinds on disposal.

Reply:
Aha, right. This could be an issue on skin change.